observability/WFLY-19846-promote-mp-tel-2-to-default.adoc

```diff
@@ -0,0 +1,115 @@
+---
+categories:
+- microprofile
+- observability
+stability-level: default
+issue: https://github.com/wildfly/wildfly-proposals/issues/619
+feature-team:
+ developer: jasondlee
+ sme:
+  -
+ outside-perspective:
+  -
+promotes: https://issues.redhat.com/browse/WFLY-19590
+promoted-by: https://github.com/wildfly/wildfly-proposals/issues/619
+---
+= [Default] Promote MicroProfile Telemetry 2.0 to WildFly Standard
+:author:            Jason Lee
+:email:             jasondlee@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+MicroProfile Telemetry 2.0 shipped with WildFly Preview 34. To meet WildFly and product roadmap goals, this feature needs to be migrated to the WildFly feature pack, and its stability level promoted to `DEFAULT`.
+
+=== User Stories
+
+User wants to deploy an application using MicroProfile Telemetry 2.0 from MicroProfile Platform 7.0.
+
+== Issue Metadata
+
+- https://issues.redhat.com/browse/WFLY-19846[]
+
+=== Related Issues
+
+- https://issues.redhat.com/browse/EAP7-2227[]
+
+=== Affected Projects or Components
+
+- MicroProfile Fault Tolerance
+- AMQP
```

Comment:
How this WF feature affects AMQP or Kafka?

Reply:
@marekkopecky I think he is referring to https://issues.redhat.com/browse/WFLY-19835 and https://issues.redhat.com/browse/WFLY-19836 which is about integrating the MP RM AMQP and Kafka connectors with Otel. That integration requires dependencies brought in via MP Telemetry 2.0, and will not work with the existing MP Telemetry 1.x deps